### DISCUSSION TITLE
assembly  genome  using WGS and RNAseq data

### DISCUSSION THREAD

**Discussion by yirenheihei** [ablab/spades#740] [Q&A]
Hi:
     I have sequenced a WGS data and fifty RNAseq data that were under different conditions for  the same strain .Can I use the WGS data and the RNAseq(using RNAspades produce the contig ) more accurately assembly my genome ?
my steps is :
fisrt: using  rnaSPAdes  assembly my fifty RNAseq into a contig.
second: using SPAdes assembly the WGS data and contig(using --truth-contig) into the final assembly genome.
now ,I want to know that Is it reasonable? or can you give a some advice?thank you ,I am kooking forward to your reply.

**Comment by asl** [comment]
Tagging @andrewprzh for help :)

**Comment by andrewprzh** [comment]
Thank you for your question.
First of all, what kind of organism are you working on? If it's eukaryote than intron may cause problems in your pipeline.

Assembly of all 50 samples together seems reasonable.
The second step could be problematic because of introns in you assembled RNA contigs. I'm not sure than using transcripts as trusted contigs will improve the assembly quality. 

I'd suggest to use SPAdes to assemble WGS data alone, and then try to use assembly improvement software, that can scaffold genome contigs using RNA assembly (however, the improvement may be marginal). For example, we have developed a package called SGTK, which has a tool for genome scaffolding using RNA data: https://github.com/olga24912/SGTK#sec5
You may give it a try.

I've seen other tools like https://github.com/mourisl/Rascaf or L_RNA_Scaffolder.

**Comment by yirenheihei** [comment]
Thank you for your reply.My organism that I am going to assembly is yeast. The species have a little intron in some genes but most of the genes have no intron.I will try the softwares that you recommended.Thank you very much again.
